include/vrv/object.h

```diff
@@ -829,36 +889,44 @@ class Object : public BoundingBox {
     /**
      * Adjust the position of the dots for multiple layers
      */
-    ///@{
     virtual int AdjustDots(FunctorParams *) { return FUNCTOR_CONTINUE; }
+
+    /**
+     * End Functor for Object::AdjustDots
+     */
     virtual int AdjustDotsEnd(FunctorParams *) { return FUNCTOR_CONTINUE; }
-    ///@}
 
     /**
      * Adjust the position of notes and chords for multiple layers
      */
-    ///@{
     virtual int AdjustLayers(FunctorParams *) { return FUNCTOR_CONTINUE; }
+
+    /**
+     * End Functor for Object::AdjustLayers
+     */
     virtual int AdjustLayersEnd(FunctorParams *) { return FUNCTOR_CONTINUE; }
-    ///@}
 
     /**
-     * @name Lay out the X positions of the grace notes looking at the bounding boxes.
+     * Lay out the X positions of the grace notes looking at the bounding boxes.
```

Comment:
You have "LayOut" as a function prefix that you've used, but this function is called "AdjustGraceXPos". Maybe for terminology you can fix the docstring? "Lay out the X positions" -> "Adjust the X postitions"?